enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -408,19 +408,19 @@ To customize the layout, use namespaced components within the `InteractiveQuesti
-| `FilterButton`          | The button used in the default layout to open the Filter pane. You can replace this button with your own implementation.     |
-| `ResetButton`           | The button used to reset the question after the question has been modified with filters/aggregations/etc                     |
-| `Title`                 | The question's title                                                                                                         |
-| `Summarize`             | The Summarize pane containing all possible aggregations                                                                      |
-| `SummarizeButton`       | The button used in the default layout to open the Summarize pane. You can replace this button with your own implementation.  |
-| `Notebook`              | The Notebook editor that allows for more filter, aggregation, and custom steps                                               |
-| `NotebookButton`        | The button used in the default layout to open the Notebook editor. You can replace this button with your own implementation. |
-| `QuestionVisualization` | The chart visualization for the question                                                                                     |
+| Component               | Info                                                                                                                        |
+|-------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+| `BackButton`            | The back button, which provides `back` functionality for the InteractiveDashboard                                           |
+| `FilterBar`             | The row of badges that contains the current filters that are applied to the question                                        |
+| `Filter`                | The Filter pane containing all possible filters                                                                             |
+| `FilterButton`          | The button used in the default layout to open the Filter pane. You can replace this button with your own implementation.    |
+| `ResetButton`           | The button used to reset the question after the question has been modified with filters/aggregations/etc                    |
+| `Title`                 | The question's title                                                                                                        |
+| `Summarize`             | The Summarize pane containing all possible aggregations                                                                     |
+| `SummarizeButton`       | The button used in the default layout to open the Summarize pane. You can replace this button with your own implementation. |
+| `Editor`                | The Query Editor that allows for more filter, aggregation, and custom steps                                                 |
```

Comment:
@jeff-bruemmer I felt weird just putting `Editor` - it felt bare and non-descriptive without a qualifier here. Should we stick with `Notebook Editor` or something else?

Reply:
I found the case we use is inconsistent. I'd expect a component to have a capitalized case like this, but on line 359 we use lowercase `query editor`.

Maybe that's expected, but it's just something that bugs me.

Reply:
no, that's good instinct, that's what keeps little bugs out of things 😄 I will change this instance to `Query Editor` but I suspect Jeff will have a better name or suggestion that we can use